extra_settings/models.py

```diff
@@ -208,7 +216,17 @@ def __init__(self, *args, **kwargs):
             self.value = value
         self.name_initial = self.name
 
+    def validate(self):
+        if not self.validator:
+            return
+        module_name, validator_func_name = str(self.validator).rsplit(".", 1)
+        module = importlib.import_module(module_name)
+        validator = getattr(module, validator_func_name)
```

Comment:
Better to rename this `validator` to `validator_func` to avoid confusion with the field name.